include/wx/qt/dcclient.h

```diff
@@ -37,6 +39,8 @@ class WXDLLIMPEXP_CORE wxClientDCImpl : public wxWindowDCImpl
 private:
     wxDECLARE_CLASS(wxClientDCImpl);
     wxDECLARE_NO_COPY_CLASS(wxClientDCImpl);
+
+    QPicture* m_pic;
```

Comment:
Maybe you can use [`wxScopedPtr<QPicture>`](https://docs.wxwidgets.org/3.1/classwx_scoped_ptr_3_01_t_01_4.html) instead of doing new/delete yourself?

Reply:
Thanks for the suggestion @MaartenBent however doing so would mean clients of the class would need to include `QScopedPointer`. They are currently completely insulated from QT and I believe it would be better to keep it this way.

Reply:
Note that Maarten wrote `wxScopedPtr`, not `QScopedPointer`.

Reply:
I still think the idea of using `wxScopedPtr` here is good. Also, I liked the old `pict` name while `pic` doesn't really associate with "picture" for me, could we please keep calling this `m_pict`?

And, as a final nitpick, could you please declare this variable above the `wxDECLARE_XXX` macros, which are supposed to be the very last thing in the class declaration?